### PR TITLE
CSS tweaks

### PR DESCRIPTION
**Pull request by MartijnCuppens**
WIP, still needs some responsive tweaks

### CODE REVIEW

File: src/css/bs-stepper.css [hunk 1]
```diff
@@ -1,198 +1,68 @@
-/*!
```
Comment (Johann-S) [inline src/css/bs-stepper.css]: Please do not remove the header

File: src/css/bs-stepper.css [hunk 2]
```diff
@@ -1,198 +1,68 @@
...
-.bs-stepper.vertical {
-  display: flex;
-}
-
-.bs-stepper .bs-stepper-header {
+.bs-stepper-btn {
+  padding: 24px;
+  font-weight: 700;
+  text-align: center;
+  text-decoration: none;
```
Comment (Johann-S) [inline src/css/bs-stepper.css]: I don't understand why did you add this new CSS class ? `.bs-stepper-btn`

**Comment by Johann-S** [comment]
The docs looks awesome ! Thank you so much 👍 😍

File: src/css/bs-stepper.css (hunk 2 above)
Comment (MartijnCuppens) [reply src/css/bs-stepper.css]: It's better to add specific classes if you want to theme an element.

**Comment by MartijnCuppens** [comment]
Everything should be ok now, I added the bootstrap js for the responsive menu. What do you think?

File: src/js/util.js [hunk 3]
```diff
@@ -10,7 +10,7 @@ const Selectors = {
 const ClassName = {
   ACTIVE: 'active',
   LINEAR: 'linear',
-  BLOCK: 'dstepper-block',
+  BLOCK: 'show',
```
Comment (Johann-S) [inline src/js/util.js]: Please revert this change, I don't want my stepper to depend on Bootstrap, I want to be able to use it without Bootstrap

File: src/css/bs-stepper.css [hunk 4]
```diff
@@ -1,198 +1,117 @@
...
-
-.bs-stepper.vertical .bs-stepper-content {
-  padding: 0 0 0 48px;
-}
-
-.bs-stepper .step {
-  display: flex;
-  height: 72px;
-  overflow: hidden;
+.bs-stepper-btn {
```
Comment (Johann-S) [inline src/css/bs-stepper.css]: I think this class isn't usefull because there is `.step-trigger` which is already on the same elements

**Comment by Johann-S** [comment]
We lost cursor `pointer` here 😟 
image

But the responsiveness is awesome thanks ! ❤️

File: README.md [hunk 5]
```diff
@@ -75,14 +75,14 @@ Add the following HTML:
   <div class="bs-stepper-header" role="tablist">
     <!-- your steps here -->
     <div class="step" data-target="#logins-part">
-      <button type="button" class="btn btn-link step-trigger" role="tab" aria-controls="logins-part" id="logins-part-trigger">
+      <button type="button" class="btn btn-link bs-stepper-btn step-trigger" role="tab" aria-controls="logins-part" id="logins-part-trigger">
```
Comment (Johann-S) [inline README.md]: we shouldn't use both `.step-trigger` and `.bs-stepper-btn` but we should be able to use those two classes

File: README.md [hunk 6]
```diff
@@ -75,14 +75,14 @@ Add the following HTML:
...
         <span class="bs-stepper-circle">1</span>
         <span class="bs-stepper-label">Logins</span>
       </button>
     </div>
     <div class="line"></div>
     <div class="step" data-target="#information-part">
-      <button type="button" class="btn btn-link step-trigger" role="tab" aria-controls="information-part" id="information-part-trigger">
+      <button type="button" class="btn btn-link bs-stepper-btn step-trigger" role="tab" aria-controls="information-part" id="information-part-trigger">
```
Comment (Johann-S) [inline README.md]: same here

File: src/css/bs-stepper.css [hunk 7]
```diff
@@ -4,195 +4,119 @@
...
```
Comment (Johann-S) [inline src/css/bs-stepper.css]: Can you add back `.step-trigger` I don't want to introduce breaking changes

**Comment by Johann-S** [comment]
Thanks for your hard work @MartijnCuppens 👍
